elastic/kibana
# [Synthetics] replace references to Uptime with Synthetics

## Summary

Resolves https://github.com/elastic/kibana/issues/150896

Replaces references to Uptime in the Elastic Synthetics integration with Synthetics

On the integration edit page: [A-params-monitor-A-private-location-default---Elastic-Synthetics---Integrations---Elastic.webm](https://user-images.githubusercontent.com/11356435/218196665-c5970d05-5c46-4181-97c6-fd772773322c.webm)
On the integrations asseets page: [Elastic-Synthetics---Integrations---Elastic.webm](https://user-images.githubusercontent.com/11356435/218196667-2dda5903-96c0-45ac-830e-d4df78675fe9.webm)
On the integrations add page: [Add-integration---Elastic-Synthetics---Integrations---Elastic.webm](https://user-images.githubusercontent.com/11356435/218196669-d29775a9-13f8-4a0c-92ae-683cd9d36cb6.webm)


### Testing
1. Create a private location in Synthetics
2. Create a monitor with that private location in Synthetics
3. Navigate to `app/integrations/detail/synthetics/policies`
4. Click on the integration policy for that app. Ensure the edit button navigates to the Synthetics edit monitor view
5. Navigate to `app/integrations/detail/synthetics/assets`. 
6. Ensure the View monitors in Synthetics button navigates to the Synthetics Overview page
7. Navigate to `app/integrations/detail/synthetics/overview`. Click Add Integration.
8. Ensure the popover references Synthetics and navigates to Synthetics.
